warehouse/query-core/src/main/java/org/apache/commons/jexl2/parser/JexlNodes.java

```diff
@@ -95,6 +95,33 @@ public static ASTReferenceExpression wrap(JexlNode node) {
         return children(ref, node);
     }
     
+    public static boolean isWrapped(JexlNode node) {
+        return id(node.jjtGetParent()) == ParserTreeConstants.JJTREFERENCEEXPRESSION;
+    }
+    
+    /**
+     * <pre>
+     * From
+     *  JexlScript -> Reference -> ReferenceExpression -> EQNode
+     * To
+     *  JexlScript -> EQNode
+     * </pre>
+     *
+     * @param node
+     * @param child
+     * @return
+     */
+    public static JexlNode unwrap(JexlNode node, JexlNode child) {
```

Comment:
is this method still used?

Reply:
It's no longer used, removing it in the next commit.